authors/apps/comments/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from .views import (
+    CommentCreateListView,CommentsListThreadsCreateView, CommentsAPIView)
+
+
+app_name = 'comments'
+urlpatterns = [
+    path('<slug>/comments/', CommentCreateListView.as_view(), name='comments'),
+    path('<slug>/comments/<int:id>', CommentsAPIView.as_view(), name='single_comment'),
+    path('<slug>/comments/<int:id>/thread', CommentsListThreadsCreateView.as_view(),name='thread comment'),
```

Comment:
Please be consistent in your naming. here you said name ='thread comment' while when I look at line 9. You say name='single_comment'. So line 10 would have been something like name='thread_comment' and therefore consistency in your naming.

Reply:
I have worked on the feedback @arthurarty